toggle/CascadingConfigParser.py

```diff
@@ -18,15 +18,15 @@
  along with Redeem.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-import ConfigParser
+import configparser
 import os
 import logging
 
 
-class CascadingConfigParser(ConfigParser.SafeConfigParser):
+class CascadingConfigParser(configparser.SafeConfigParser):
```

Comment:
Are you seeing any warnings that SafeConfigParser is deprecated?
I saw them when I started poking at 2to3 in Redeem, but if this works without warnings we can address it later?